display/ssd1306.h

```diff
@@ -506,6 +553,8 @@ class SSD1306 : public I2CDevice, Looper, StateMachine, SaberBase, private Audio
       } else {
 	looped_frames_ = height / 128;
       }
+      static_on_ = on_played_ && looped_frames_ == 1;
```

Comment:
The current code sets static_on_ to false if on_played_ is false, which I don't think is what you want.

Try something like:
    if (current_effect_ == &IMG_on) {
       looped_on_ = looped_frames_ > 1;
    }